src/WebJobs.Script/Eventing/JobHost/FunctionIndexingEvent.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
nit: there's already a Host directory - does this belong there rather than a new JobHost directory?